index.html

```diff
@@ -1676,15 +1697,19 @@ <h2 id="AudioContext">
               <li>Set a <code>control thread state</code> to
               <code>suspended</code> on the <a>AudioContext</a>.
               </li>
-              <li>Send a <a>control message</a> to start processing.
+              <li>Let <dfn>pendingResumePromises</dfn> be an empty ordered list
+              of promises.
+              </li>
+              <li>If the <a>AudioContext</a> is <a>allowed to start</a>, send a
```

Comment:
I think you also need to describe here what happens if the context is not allowed to start. Presumably all steps are aborted here.